sites/all/modules/custom/chabadsuite/cs_core/cs_core.install

```diff
@@ -63,3 +63,11 @@ function cs_core_update_7005() {
 function cs_core_update_7006() {
   module_enable(array('civicrm_cache_clear'));
 }
+
+/**
+ * Enable new features and adminimal theme
+ */
+function cs_core_update_7007() {
+  module_enable(array('cs_core_roles', 'cs_user_permission', 'cs_civi', 'cs_civi_drupal_user', 'cs_theme'));
+  theme_enable(array('adminimal'));
```

Comment:
@yosefromano fyi, I added the enabling of adminimal here, since we're creating that new admin page. Might as well have that in place so we can see what it'll actually look like.